This patch is needed due to our mips64 build uses softfloat. Qemu will not
build without this patch.

diff --git a/tests/fp/fp-bench.c b/tests/fp/fp-bench.c
index 8ce0ca154..41198adcd 100644
--- a/tests/fp/fp-bench.c
+++ b/tests/fp/fp-bench.c
@@ -553,6 +553,7 @@ void die_host_rounding(enum rounding rounding)
     exit(EXIT_FAILURE);
 }
 
+#ifndef __mips_soft_float
 static void set_host_precision(enum rounding rounding)
 {
     int rhost;
@@ -581,6 +582,7 @@ static void set_host_precision(enum rounding rounding)
         die_host_rounding(rounding);
     }
 }
+#endif
 
 static void set_soft_precision(enum rounding rounding)
 {
@@ -672,9 +674,11 @@ static void parse_args(int argc, char *argv[])
 
     /* set precision and rounding mode based on the tester */
     switch (tester) {
+#ifndef __mips_soft_float
     case TESTER_HOST:
         set_host_precision(rounding);
         break;
+#endif
     case TESTER_SOFT:
         set_soft_precision(rounding);
         switch (precision) {
